src/pages/EventPage/EventPage.tsx

```diff
@@ -75,18 +84,23 @@ function EventPage({ eventId, navigate, ...rest }: EventPageProps) {
                     location={location}
                     owner={owner}
                   />
-                  <RsvpButton event={event} onSuccess={refreshEvent} />
-                  <DeleteButton event={event} onDelete={handleDelete} />
-                  {!isNotOwner && (
-                    <Button
-                      variant="highlight"
-                      onClick={() =>
-                        navigate!(`/update-event/${event.id}/${name}`)
-                      }
-                    >
-                      Edit
-                    </Button>
-                  )}
+                  <ButtonContainer>
+                    <RsvpButton event={event} onSuccess={refreshEvent} />
+                    <DeleteButton event={event} onDelete={handleDelete} />
+                    {!isNotOwner && (
```

Comment:
I see that this wasn't written by you, but double negation is not ideal as it requires more cognitive load - was it me who wrote it 🙈?. Maybe change `isNotOwner` into `isOwner` and negate it where necessary instead 😝

Reply:
It seems it is always negated, so ill just change it to `isOwner`